chpl/chpl-service/src/main/java/gov/healthit/chpl/app/statistics/SummaryStatistics.java

```diff
@@ -121,8 +121,10 @@ public static void main(final String[] args) throws Exception {
             files.add(csvFile);
         }
         String htmlMessage = summaryStats.createHtmlMessage(emailBodyStats);
+        LOGGER.info(htmlMessage);
 
         // send the email
+        /*
```

Comment:
Was this moved to the quartz job? If so, it shouldn't be here

Reply:
File should never have been changed. It has been reverted.